irods_client/collection_operations.py

```diff
@@ -2,21 +2,26 @@
 import json
 
 class Collections:
-    # Initializes Collections with variables from the parent class.
+    """" 
+    Initializes Collections with a base url. 
+    Token is set to None initially, and updated when setToken() is called in irodsClient.
+    """
     def __init__(self, url_base: str):
         self.url_base = url_base
         self.token = None
 
 
-    # def _setToken(self, token):
-    #     self.token = token
+    """
+    Creates a new collection.
 
-    # Creates a new collection
-    # params
-    # - lpath: The absolute logical path of the collection to be created.
-    # - create_intermediates (optional): Set to 1 to create intermediates, otherwise set to 0. Defaults to 0.
-    # returns
-    # - Status code and response message.
+    Parameters
+    - lpath: The absolute logical path of the collection to be created.
+    - create_intermediates (optional): Set to 1 to create intermediates, otherwise set to 0. Defaults to 0.
+
+    Returns
+    - 2XX: Status code and iRODS response.
+    - Other: Status code and iRODS error information.
+    """
```

Comment:
I believe these are supposed to be inside the function?

@alanking Thoughts?

Reply:
I looked online and from what I can tell they are. I can go ahead and move them

Reply:
They are what? Allowed to be above functions? Is that what you read online?

Reply:
They are supposed to be inside the function they are describing.

Reply:
Cool cool.